src/app/layout.tsx

```diff
@@ -32,8 +32,8 @@ export default function RootLayout({
     >
       <body>
         <ThemeProvider
-          defaultTheme="light"
-          enableSystem={false}
+          defaultTheme="system"
```

Comment:
You can just get rid of defaultTheme and enableSystem. By default, next-theme will use system theme.